### PR TITLE
seabios: disable sga device in rhel9

### PR DESCRIPTION
**Pull request by XueqiangWei**
1. since sga device has been removed in rhel9, disable it
2. using "-machine graphics=off" instead of "-device sga"

Signed-off-by: Xueqiang Wei <xuwei@redhat.com>
ID: 2008574

### CODE REVIEW

**Comment by XueqiangWei** [comment]
case boot_order_check:
 (1/3) Host_RHEL.m8.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.boot_order_check.bootorder0.q35: PASS (233.33 s)
 (2/3) Host_RHEL.m8.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.boot_order_check.bootorder1.q35: PASS (233.52 s)
 (3/3) Host_RHEL.m8.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.boot_order_check.bootorder2.q35: PASS (237.10 s)

 (1/3) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.boot_order_check.bootorder0.q35: PASS (233.06 s)
 (2/3) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.boot_order_check.bootorder1.q35: PASS (232.50 s)
 (3/3) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.boot_order_check.bootorder2.q35: PASS (237.07 s)

case  boot_from_device:
(01/16) Host_RHEL.m8.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.boot_from_device.cdrom.ahci_cd.with_bootindex.q35: PASS (59.30 s)
 (02/16) Host_RHEL.m8.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.boot_from_device.cdrom.ahci_cd.with_specify_device.q35: PASS (30.50 s)
 (03/16) Host_RHEL.m8.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.boot_from_device.cdrom.scsi_cd.with_bootindex.q35: PASS (59.21 s)
 (04/16) Host_RHEL.m8.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.boot_from_device.cdrom.scsi_cd.with_specify_device.q35: PASS (32.45 s)
 (05/16) Host_RHEL.m8.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.boot_from_device.ahci_disk.with_bootindex.q35: PASS (33.83 s)
 (06/16) Host_RHEL.m8.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.boot_from_device.ahci_disk.with_specify_device.q35: PASS (57.04 s)
 (07/16) Host_RHEL.m8.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.boot_from_device.virtio_blk_disk.with_bootindex.q35: PASS (58.78 s)
 (08/16) Host_RHEL.m8.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.boot_from_device.virtio_blk_disk.with_specify_device.q35: PASS (57.03 s)
 (09/16) Host_RHEL.m8.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.boot_from_device.scsi_hd_disk.with_bootindex.q35: PASS (58.89 s)
 (10/16) Host_RHEL.m8.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.boot_from_device.scsi_hd_disk.with_specify_device.q35: PASS (32.28 s)
 (11/16) Host_RHEL.m8.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.boot_from_device.usb_storage_disk.usb_uhci.with_bootindex.q35: PASS (58.85 s)
 (12/16) Host_RHEL.m8.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.boot_from_device.usb_storage_disk.usb_uhci.with_specify_device.q35: PASS (32.17 s)
 (13/16) Host_RHEL.m8.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.boot_from_device.usb_storage_disk.usb_ehci.with_bootindex.q35: PASS (58.75 s)
 (14/16) Host_RHEL.m8.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.boot_from_device.usb_storage_disk.usb_ehci.with_specify_device.q35: PASS (32.17 s)
 (15/16) Host_RHEL.m8.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.boot_from_device.usb_storage_disk.usb_xhci.with_bootindex.q35: PASS (58.71 s)
 (16/16) Host_RHEL.m8.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.boot_from_device.usb_storage_disk.usb_xhci.with_specify_device.q35: PASS (32.22 s)

(01/16) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.boot_from_device.cdrom.ahci_cd.with_bootindex.q35: PASS (92.28 s)
 (02/16) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.boot_from_device.cdrom.ahci_cd.with_specify_device.q35: PASS (56.80 s)
 (03/16) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.boot_from_device.cdrom.scsi_cd.with_bootindex.q35: PASS (31.76 s)
 (04/16) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.boot_from_device.cdrom.scsi_cd.with_specify_device.q35: PASS (45.51 s)
 (05/16) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.boot_from_device.ahci_disk.with_bootindex.q35: PASS (45.51 s)
 (06/16) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.boot_from_device.ahci_disk.with_specify_device.q35: PASS (45.08 s)
 (07/16) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.boot_from_device.virtio_blk_disk.with_bootindex.q35: PASS (45.60 s)
 (08/16) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.boot_from_device.virtio_blk_disk.with_specify_device.q35: PASS (45.49 s)
 (09/16) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.boot_from_device.scsi_hd_disk.with_bootindex.q35: PASS (45.53 s)
 (10/16) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.boot_from_device.scsi_hd_disk.with_specify_device.q35: PASS (45.04 s)
 (11/16) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.boot_from_device.usb_storage_disk.usb_uhci.with_bootindex.q35: PASS (47.68 s)
 (12/16) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.boot_from_device.usb_storage_disk.usb_uhci.with_specify_device.q35: PASS (30.92 s)
 (13/16) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.boot_from_device.usb_storage_disk.usb_ehci.with_bootindex.q35: PASS (45.70 s)
 (14/16) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.boot_from_device.usb_storage_disk.usb_ehci.with_specify_device.q35: PASS (45.08 s)
 (15/16) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.boot_from_device.usb_storage_disk.usb_xhci.with_bootindex.q35: PASS (47.61 s)
 (16/16) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.boot_from_device.usb_storage_disk.usb_xhci.with_specify_device.q35: PASS (30.82 s)

case seabios_order_once
(1/3) Host_RHEL.m8.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.seabios_order_once.with_order.q35: PASS (163.44 s)
 (2/3) Host_RHEL.m8.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.seabios_order_once.with_once.q35: PASS (163.46 s)
 (3/3) Host_RHEL.m8.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.seabios_order_once.with_order_once.q35: PASS (166.64 s)

(1/3) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.seabios_order_once.with_order.q35: PASS (162.84 s)
 (2/3) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.seabios_order_once.with_once.q35: PASS (163.18 s)
 (3/3) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.seabios_order_once.with_order_once.q35: PASS (166.02 s)

case seabios_strict
 (1/2) Host_RHEL.m8.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.seabios_strict.on.q35: PASS (456.88 s)
 (2/2) Host_RHEL.m8.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.seabios_strict.off.q35: PASS (99.67 s)

 (1/2) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.seabios_strict.on.q35: PASS (455.97 s)
 (2/2) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.seabios_strict.off.q35: PASS (98.69 s)

case seabios_hotplug_unplug:
(1/1) Host_RHEL.m8.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.seabios_hotplug_unplug.q35: PASS (372.28 s)

(1/1) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.seabios_hotplug_unplug.q35: PASS (462.54 s)

case seabios:
 (1/2) Host_RHEL.m8.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.seabios.basic.q35: PASS (77.21 s)
 (2/2) Host_RHEL.m8.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.seabios.bin_file.q35: PASS (532.73 s)

 (1/2) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.seabios.basic.q35: PASS (75.87 s)
 (2/2) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.seabios.bin_file.q35: PASS (227.91 s)

**Comment by XueqiangWei** [comment]
@zhencliu Please help review it. Many thanks.

File: qemu/tests/cfg/boot_order_check.cfg [hunk 1]
```diff
@@ -6,6 +6,9 @@
     kill_vm = yes
     boot_menu = on
     enable_sga = yes
+    Host_RHEL.m9:
```
Comment (zhencliu) [inline qemu/tests/cfg/boot_order_check.cfg:9]: what about defining 
enable_sga = no
machine_type_extra_params = "graphics=off"

then for the previous rhel releases:
HOST_RHEL.m7, m8...
    enable_sga = yes

So when 10/11 comes out, we don't have to add them

Comment (XueqiangWei) [reply qemu/tests/cfg/boot_order_check.cfg:9]: Thanks for your quickly review. sga not only used for rhel host  but also other linux hosts. It just removed in rhel9, so I just disable it in rhel9. I think we have to add them when 10/11 comes out. Thanks.

**Comment by zhencliu** [comment]
LGTM

**Pull request merged**